lib/OpenQA/WebAPI/Controller/API/V1/JobGroup.pm

```diff
@@ -7,6 +7,8 @@ use Mojo::JSON;
 use Feature::Compat::Try;
 
 use OpenQA::Schema::Result::JobGroups;
+use OpenQA::Shared::Controller::OpenAPI;
+*catchall = \&OpenQA::Shared::Controller::OpenAPI::catchall;
```

Comment:
Maybe we can just inherit from `OpenQA::Shared::Controller::OpenAPI`.

Reply:
I tried, but somehow it didn't work, but I don't know the error message any more